doc/devel/release_guide.rst

```diff
@@ -159,7 +190,7 @@ Update release notes
 What's new
 ^^^^^^^^^^
 
-*Only needed for major and minor releases. Bugfix releases should not have new
+*Only needed for major and meso releases. Bugfix releases should not have new
```

Comment:
Here and below, too: do you want to leave "major", or switch to "macro"?

Reply:
the limitations find-and-replace and insufficient proof reading 😞